addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/handler/LoxoneMiniserverHandler.java

```diff
@@ -0,0 +1,574 @@
+                    if ((OnOffType) command == OnOffType.ON) {
+                        if (control instanceof LxControlPushbutton) {
+                            ((LxControlPushbutton) control).pulse();
+                        } else {
+                            ((LxControlSwitch) control).on();
+                        }
+                    } else {
+                        ((LxControlSwitch) control).off();
+                    }
+                }
+                return;
+            }
+
+            if (control instanceof LxControlJalousie) {
+                LxControlJalousie jalousie = (LxControlJalousie) control;
+                if (command instanceof PercentType) {
+                    jalousie.moveToPosition(((PercentType) command).doubleValue() / 100);
+                } else if (command instanceof UpDownType) {
+                    if ((UpDownType) command == UpDownType.UP) {
+                        jalousie.fullUp();
+                    } else {
+                        jalousie.fullDown();
+                    }
+                } else if (command instanceof StopMoveType) {
+                    if ((StopMoveType) command == StopMoveType.STOP) {
+                        jalousie.stop();
+                    }
+                }
+                return;
+            }
+
+            if (control instanceof LxControlLightController) {
+                LxControlLightController controller = (LxControlLightController) control;
+                if (command instanceof OnOffType) {
+                    if ((OnOffType) command == OnOffType.ON) {
+                        controller.allOn();
+                    } else {
+                        controller.allOff();
+                    }
+                } else if (command instanceof UpDownType) {
+                    if ((UpDownType) command == UpDownType.UP) {
+                        controller.nextScene();
+                    } else {
+                        controller.previousScene();
+                    }
+                } else if (command instanceof DecimalType) {
+                    controller.setScene(((DecimalType) command).intValue());
+                }
+                return;
+            }
+
+            if (control instanceof LxControlRadio) {
+                LxControlRadio radio = (LxControlRadio) control;
+                if (command instanceof OnOffType) {
+                    if ((OnOffType) command == OnOffType.OFF) {
+                        radio.setOutput(0);
+                    }
+                } else if (command instanceof DecimalType) {
+                    radio.setOutput(((DecimalType) command).intValue());
+                }
+                return;
+            }
+
+            logger.debug("Incompatible operation on control {}", control.getUuid().toString());
+
+        } catch (IOException e) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
+        } catch (Exception e) {
```

Comment:
Given that it is not the main functionality, I can follow your arguments. In the general case I personally rather fail and make that clear to the user then kind of ignoring an error of which we not even know what is going on.